chanfitness/revenue.go

```diff
@@ -21,6 +21,21 @@ var (
 type eventsQuery func(
 	offset, maxEvents uint32) (channeldb.ForwardingLogTimeSlice, error)
 
+// GetRevenue calculates accumulates relevant forwarding logs and calcualtes
+// revenue for a target channel.
+func GetRevenue(channelID lnwire.ShortChannelID,
```

Comment:
Typically we like to accept channel points where possible as they're more immediately queryable (ask your node for example). The forwarding log is based on channel ID, but there're utility methods to obtain the channel ID that maps to a channel point. There're several possibilities, but in practice we can ignore most of them.

Reply:
Actually see this method: https://godoc.org/github.com/lightningnetwork/lnd/lnwire#ChannelID.IsChanPoint